parlai/core/teachers.py

```diff
@@ -1611,12 +1611,12 @@ def setup_data(self, datafile):
                 yield act, next_episode_new
 
 
-class ConversationTeacher(FixedDialogTeacher):
+class ConversationTeacher(ParlAIDialogTeacher):
```

Comment:
I think it might need to be DialogTeacher, not ParlAIDialogTeacher

Reply:
What makes `DialogTeacher` preferable to `ParlAIDialogTeacher` here?

Reply:
ParlAI DT should be assuming a particular format. It might be that we successfully override everything but really we shouldn't.